eng/pipelines/fluent_java_mgmt_docs.yml

```diff
@@ -0,0 +1,47 @@
+trigger: none #this is not a CI, the pipeline will be triggered manually
+
+variables: 
+- group: 'GitHub Release Variables' 
```

Comment:
Can you please remove this from the yml we try an avoid defining these in yml and only hook them up in the web ui.